File: python3/_programmers/Hash/42576.py
def solution(participant, completion):
    filter = {}
    for man in participant:
        filter[man] = 0
    for man in participant:
        filter[man] += 1
    for man in completion:
        filter[man] -= 1
    for man in set(participant):
        if filter[man] >= 1:
            return man
    return None

File: cpp/programmers/level1/42576.cpp
#include <bits/stdc++.h>
#define fastio cin.tie(0)->sync_with_stdio(0)

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    sort(participant.begin(), participant.end());
    sort(completion.begin(), completion.end());
    int s = participant.size();
    for (int i = 0; i < s; ++i) {
        if (participant[i] != completion[i]) return participant[i];
    }
    return "";
}